package.json

```diff
@@ -87,5 +87,13 @@
     "tsconfig-paths": "4.2.0",
     "typescript": "5.5.4",
     "node-dir": "^0.1.17"
+  },
+  "dependencies": {
+    "@types/eventsource": "^1.1.15",
+    "@types/lodash": "^4.17.7",
+    "eventsource": "^2.0.2",
+    "lodash": "^4.17.21",
```

Comment:
do not install lodash, you can just install the exact dep https://www.npmjs.com/package/lodash.chunk